engine/camunda/src/main/java/org/camunda/bpm/delegate/UserTaskEntry.java

```diff
@@ -5,25 +5,125 @@
 import kong.unirest.Unirest;
 import org.camunda.bpm.engine.delegate.DelegateTask;
 import org.camunda.bpm.engine.delegate.TaskListener;
+import org.camunda.bpm.model.bpmn.BpmnModelInstance;
+import org.camunda.bpm.model.bpmn.instance.BoundaryEvent;
+import org.camunda.bpm.model.bpmn.instance.Documentation;
+import org.camunda.bpm.model.bpmn.instance.Message;
+import org.camunda.bpm.model.bpmn.instance.MessageEventDefinition;
+import org.camunda.bpm.model.bpmn.instance.UserTask;
 
 import java.io.IOException;
+import java.util.Collection;
 import java.util.Properties;
+import java.util.logging.Logger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class UserTaskEntry implements TaskListener {
+    private static final Logger LOGGER = Logger.getLogger(UserTaskEntry.class.getName());
 
     @Override
     public void notify(DelegateTask delegateTask) {
-
         Properties properties = new Properties();
         try {
             properties.load(getClass().getClassLoader().getResourceAsStream("config.properties"));
+           
         } catch (IOException e) {
-            e.printStackTrace();
+            LOGGER.severe("Unable to load config file: " + e.getMessage());
+        }
+
+        BpmnModelInstance bpmModel = delegateTask.getProcessEngineServices().getRepositoryService()
+                .getBpmnModelInstance(delegateTask.getProcessDefinitionId());
+        UserTask userTask = (UserTask) bpmModel.getModelElementById(delegateTask.getTaskDefinitionKey());
+
+        Collection<BoundaryEvent> boundaryEvents = bpmModel.getModelElementsByType(BoundaryEvent.class);
+        LOGGER.info("Total Boundary Events in model: " + boundaryEvents.size());

```

Comment:
Developer debug information is not desired in log files

Reply:
@PatrickBonne  I understand and I will take care of the logfiles,